scapy/contrib/automotive/uds.py

```diff
@@ -122,6 +121,10 @@ class UDS_DSC(Packet):
         ByteEnumField('diagnosticSessionType', 0, diagnosticSessionTypes)
     ]
 
+    def get_log(self):
```

Comment:
This sounds very similar to `mysummary()`

Reply:
That's right. I'll change it. Thanks for this hint.

Reply:
I double checked `mysummary`. Since it delivers information for `summary` I wouldn't like to override it. Some of my `get_log` methods produce quite a lot of binary gibberish. This would mess up the `summary` completely.